src/current/v24.1/molt-fetch.md

```diff
@@ -275,23 +309,106 @@ To drop existing tables and create new tables before loading the data, use `'dro
+
+	- [`TIMESTAMP`]({% link {{ page.version.version }}/timestamp.md %}) <> [`TIMESTAMPTZ`]({% link {{ page.version.version }}/timestamp.md %})
+	- [`VARCHAR`]({% link {{ page.version.version }}/string.md %}) <> [`UUID`]({% link {{ page.version.version }}/uuid.md %})
+	- [`BOOL`]({% link {{ page.version.version }}/bool.md %}) <> [`INT2`]({% link {{ page.version.version }}/int.md %})
+	- [`VARBIT`]({% link {{ page.version.version }}/bit.md %}) <> [`TEXT`]({% link {{ page.version.version }}/string.md %})
+	- [`JSONB`]({% link {{ page.version.version }}/jsonb.md %}) <> [`TEXT`]({% link {{ page.version.version }}/string.md %})
+	- [`INET`]({% link {{ page.version.version }}/inet.md %}) <> [`TEXT`]({% link {{ page.version.version }}/string.md %})
+
+`--type-map-file` specifies the path to the JSON file that contains explicit type mappings. For example:
+
+{% include_cached copy-clipboard.html %}
+~~~
+--type-map-file 'type-mappings.json'
+~~~
+
+The JSON should be formatted as follows:
+
+~~~ json
+[
+  {
+    "table": "public.t1",
+    "column-type-map": [
+      {
+        "column": "*",
+        "type-kv": {
+          "source-type": "int",
+          "crdb-type": "INT2"
+        }
+      },
+      {
+        "column": "name",
+        "type-kv": {
+          "source-type": "varbit",
+          "crdb-type": "string"
+        }
+      }
+    ]
+  }
+]
+~~~
+
+- `type-kv` specifies the `source-type` that maps to the target `crdb-type`.
+- `column` specifies the column name to convert. If `*` is specified, then all columns with the matching `source-type` are converted.
 
 ### Fetch continuation
 
-If `molt fetch` exits with an error after loading data from [cloud](#cloud-storage) or [local storage](#local-file-server), you can continue the process from the *continuation point* where it was interrupted.
+If `molt fetch` fails while loading data into CockroachDB from intermediate files, it exits with an error message, fetch ID, and [continuation token](#list-active-continuation-tokens) for each table that failed to load on the target database. You can use this information to continue the process from the *continuation point* where it was interrupted. For an example, see [Continue fetch after encountering an error](#continue-fetch-after-encountering-an-error).
+
+Continuation is only possible under the following conditions:
+
+- All data has been exported from the source database into intermediate files on [cloud](#cloud-storage) or [local storage](#local-file-server).
+- The *initial load* of source data to the target CockroachDB database is incomplete. This means that ongoing [replication](#replication) of source data has not begun.
 
-To retry all data starting from the continuation point, include `--fetch-id` and specify the process ID from the `molt fetch` output.
+{{site.data.alerts.callout_info}}
+Only one fetch ID and set of continuation tokens are active at any time. See [List active continuation tokens](#list-active-continuation-tokens).
```

Comment:
Nit: `set of corresponding continuation tokens`. I don't know if it's worth mentioning, but each continuation token maps to a single table